clients/unshaded/src/main/java/tachyon/client/file/FileSystemContext.java

```diff
@@ -72,4 +72,8 @@ public synchronized void reset() {
     mFileSystemMasterClientPool =
         new FileSystemMasterClientPool(ClientContext.getMasterAddress());
   }
+
+  public static FileSystemContext get() {
```

Comment:
Added to make it easier to mock places that would use FileSystemContext.INSTANCE. They can instead use FileSystemContext.get(), which can be mocked.